### PR TITLE
Login

### PR DESCRIPTION
**Pull request by ReynardEra**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,19 @@
 import React, { Component } from 'react';
-import { Route, Link } from 'react-router-dom'
+import { Route } from 'react-router-dom'
 import './css/index.css';
-import { headerNav } from './header.js'
```
Comment (KimberleighBeyl) [inline src/App.js]: HeaderNav -> remember  capitalization

File: src/App.js [hunk 2]
```diff
@@ -15,14 +23,18 @@ class App extends Component {
...
-          <headerNav></headerNav>
-          <Link to="/">Home</Link>
+
         </header>
 
         <main>
-          <Route exact path="/" component={home} />
-        </main>
+            <Route exact path="/" component={home} />
+            <Route exact path="/nav" component={headerNav} />
```
Comment (KimberleighBeyl) [inline src/App.js]: headerNav should be HeaderNav -> remember capitalization

File: src/reducers/index.js [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-import reducers1 from './reducer1'
+import reducers1 from './reducer1';
 
 export default combineReducers({
-  routing: routerReducer,
-  // TODO : register all the imported reducers
-  reducers1
-})
+    routing: routerReducer,
+    // TODO : register all the imported reducers
```
Comment (wscheffer) [inline src/reducers/index.js]: Is this comment still valid?

File: src/App.js (hunk 2 above)
Comment (ReynardEra) [reply src/App.js]: Removed it. was conflicting with other stuff

File: src/App.js [hunk 4]
```diff
@@ -1,11 +1,19 @@
 import React, { Component } from 'react';
-import { Route, Link } from 'react-router-dom'
+import { Route } from 'react-router-dom'
 import './css/index.css';
-import { headerNav } from './header.js'
+
+
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+//Import Material-UI
```
[the remark this answers is not in the document]
Comment (ReynardEra) [reply src/App.js]: Fixed the ghost code

File: src/App.js [hunk 5]
```diff
@@ -1,11 +1,19 @@
...
 
 // TODO : Import all the components
```
[the remark this answers is not in the document]
Comment (ReynardEra) [reply src/App.js]: Removed comments not valid

File: src/Login/Login_index.css [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+  height: 72px !important;
+  display: inline-block !important;
+  position: relative !important;
+  background-color: transparent !important;
+  font-family: Helvetica !important;
+  transition: height 200ms cubic-bezier(0.23, 1, 0.32, 1) 0ms !important;
+  left: 71px;
+}
+
+.loginbtn {
```
[the remark this answers is not in the document]
Comment (ReynardEra) [reply src/Login/Login_index.css]: Fixed indentations

File: src/reducers/index.js (hunk 3 above)
Comment (ReynardEra) [reply src/reducers/index.js]: Removed invalid comment

**Pull request closed without merging**